bullet/src/SDFFeatures.cc

```diff
@@ -60,17 +60,20 @@ Identity SDFFeatures::ConstructSdfLink(
       convertVec(ignition::math::eigen3::convert(diagonalMoments));
 
   // (TO-DO: do those calculation need to take into account in some way the base model?)
```

Comment:
Can you remove this comment? I think you addressed this out here

Reply:
Done on a4c0c69